backend/services/prisonerProfileService.js

```diff
@@ -129,10 +131,7 @@ module.exports = ({
     const useOfForceEnabledPrisons = useOfForcePrisons.split(',').map(prison => prison.trim().toUpperCase())
 
     const isSocUser = Boolean(
-      userRoles &&
-        userRoles.some(role =>
-          ['SOC_CUSTODY', 'SOC_COMMUNITY', 'SOC_EXTERNAL_RO', 'SOC_EXTERNAL'].includes(role.roleCode)
```

Comment:
Also tidied up these roles whilst I was here, 'SOC_EXTERNAL_RO' and 'SOC_EXTERNAL' no longer exist